### ISSUE TITLE
release kernel source for mi5s plus

### ISSUE THREAD

**Issue by anetar** [MiCode/Xiaomi_Kernel_OpenSource#411]
please release kernel source for mi5s plus

**Comment by paranoiagu** [comment]
please release kernel source for mi5s plus

**Comment by anetar** [comment]
Hi, the N kernel source code is live at: https://github.com/OnePlusOSS/androi...eplus/QC8996_N

**Comment by Grief** [comment]
Cannot @torvalds help us?

**Comment by blackjetcat** [comment]
Now you can close issue, source released ( scorpio-m-oss ).

**Comment by Grief** [comment]
@sh0w0ffin are you high? scorpio is mi note 2, not mi5s plus!

**Comment by luk1337** [comment]
@Grief Are you high? look at the commit message, it's common kernel for all 8996 devices :^)

**Comment by Grief** [comment]
@luk1337 @sh0w0ffin yes, I am. Didn't see the actual commit message 'Kernel: Xiaomi kernel changes for Mi 5/Mi Note2/Mi 5s/Mi 5s plus/Mi Mix '. Sorry about that. Congratulations then!

**Comment by dksslq** [comment]
Can "scorpio-m-oss" compile and repackage it into boot.img and then flash it into Xiaomi 5s plus?,I am new.....

**Comment by luk1337** [comment]
@dksslq no one stops you from doing that :3
